Author: Emik Lin (HKUMed CPOS)
Use the pinned local libBigWig source instead of cloning it during the build.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -121,9 +121,8 @@
 # TODO: separate source and binary dir?
 ExternalProject_Add(libBigWig
-    GIT_REPOSITORY "https://github.com/cpockrandt/libBigWig"
+    DOWNLOAD_COMMAND ""
     UPDATE_COMMAND ""
-    DOWNLOAD_DIR "${CMAKE_SOURCE_DIR}/external/"
-    SOURCE_DIR "${CMAKE_BINARY_DIR}/libBigWig"
+    SOURCE_DIR "${CMAKE_SOURCE_DIR}/../libBigWig-f390fe405dede0a4f9e8e2a49808e3d1bfcd402a"
     BUILD_IN_SOURCE 1
     CONFIGURE_COMMAND ""
     BUILD_COMMAND "${LIBBIGWIG_MAKE_CMD}"
@@ -154,2 +153 @@
-include_directories(SYSTEM ${CMAKE_BINARY_DIR}/libBigWig)
-link_directories(${CMAKE_BINARY_DIR}/libBigWig)
+include_directories(SYSTEM ${CMAKE_SOURCE_DIR}/../libBigWig-f390fe405dede0a4f9e8e2a49808e3d1bfcd402a)
@@ -177 +175 @@
-target_link_libraries(phylocsf++ BigWig)
+target_link_libraries(phylocsf++ "${CMAKE_SOURCE_DIR}/../libBigWig-f390fe405dede0a4f9e8e2a49808e3d1bfcd402a/libBigWig.a")
